README.md

```diff
@@ -39,7 +39,7 @@ See our [Getting Started Guide](https://docs.cipherstash.com/tutorials/getting-s
 Add this line to your applications Gemfile:
 
 ```ruby
-gem 'activestash'
+gem 'active_stash'
```

Comment:
Fixing up the gem name while I'm in the area. I noticed that I need to use `'active_stash'` instead of `'activestash'` while testing in a local rails app.